server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/QualityGatesWsSupport.java

```diff
@@ -92,12 +93,22 @@ OrganizationDto getOrganization(DbSession dbSession, Request request) {
     return checkFoundWithOptional(organizationDto, "No organization with key '%s'", organizationKey);
   }
 
+  /**
+   * @deprecated use {@link #checkCanEdit(QGateWithOrgDto)} instead
+   */
+  @Deprecated
   void checkCanEdit(QualityGateDto qualityGate) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)